misc/services/mobile-webui/mobile-webui-frontend/src/utils/toast.jsx

```diff
@@ -32,16 +32,12 @@ export const toastError = ({ axiosError, messageKey, fallbackMessageKey, plainMe
   }
 
   console.trace('toast error: ', { message, axiosError, context });
-  toast.custom(
-    (t) => (
-      <div className="toastContainer" onClick={() => toast.dismiss(t.id)}>
-        <span>{message}</span>
-      </div>
-    ),
-    {
-      duration: 86400000,
-    }
-  );
+
+  toast.error(message, {
```

Comment:
what's the actual impact on how the toasts are displayed?

Reply:
![image](https://github.com/user-attachments/assets/a893c28b-147c-4e9a-94f4-9a3e62fa454f)